os-hardening/templates/limits.conf.tmpl

```diff
@@ -1,4 +1,4 @@
-# {{ pillar['message_do_not_modify'] }}
+# {{ salt['pillar.get']('message_do_not_modify','MANAGED BY SALT - DO NOT MODIFY') }}
```

Comment:
I removed it so the message is just static. I think the previous case was a slight overuse of pillar variables (it's quite rare to want  a custom header message in the managed config files, and not a feature usually provided by formulas)